samples/spring-client-java-valueentity-counter/src/main/java/com/example/client/spring/rest/CounterClientApplication.java

```diff
@@ -0,0 +1,11 @@
+package com.example.client.spring.rest;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+
+@SpringBootApplication
+public class CounterClientApplication {
+    public static void main(String[] args) {
```

Comment:
Although the Java community tends to use 4 spaces indentation, we prefer to use 2 spaces. The main reason is that it takes less space and when we add code snippets in the docs, we want it to be little more compact.

Reply:
Updated indentation.